pyro/infer/svi.py

```diff
@@ -51,15 +51,6 @@ def _loss_and_grads(*args, **kwargs):
             self.loss = loss
             self.loss_and_grads = loss_and_grads
 
-    def __call__(self, *args, **kwargs):
```

Comment:
why is this being removed here?

Reply:
We don't want to support this API, and I remembered it while going over the tutorial